### DISCUSSION TITLE
Multiple Space Support

### DISCUSSION THREAD

**Discussion by warengonzaga** [anyproto/contributors#16] [Q&A] [answered]
I'm currently using Anytype, and would like to know if it is possible in the future to support multiple personal space account?

**Comment by SamBouwer** [comment]
You can already create multiple accounts with one space each, but I suspect you want to access multiple personal spaces under the same account. I'm quite sure this will become possible with shared spaces as I expect it is possible to just share such a space with yourself :)

**Comment by isle9** [comment, marked as answer]
It's already on the [roadmap](https://github.com/anyproto/roadmap/issues/23), right?

**Reply by warengonzaga** [reply]
Perfect!

**Reply by ra3orblade** [reply]
its in development atm
